executor/write_test.go

```diff
@@ -356,12 +356,22 @@ func (s *testSuite) TestInsertIgnore(c *C) {
 	r.Check(testkit.Rows(rowStr))
 
 	tk.MustExec("insert ignore into t values (1, 3), (2, 3)")
-
 	r = tk.MustQuery("select * from t;")
-	rowStr = fmt.Sprintf("%v %v", "1", "2")
 	rowStr1 := fmt.Sprintf("%v %v", "2", "3")
 	r.Check(testkit.Rows(rowStr, rowStr1))
 
+	tk.MustExec("insert ignore into t values (3, 4), (3, 4)")
+	r = tk.MustQuery("select * from t;")
+	rowStr2 := fmt.Sprintf("%v %v", "3", "4")
+	r.Check(testkit.Rows(rowStr, rowStr1, rowStr2))
+
+	tk.MustExec("begin")
+	tk.MustExec("insert ignore into t values (4, 4), (4, 5), (4, 6)")
+	r = tk.MustQuery("select * from t;")
+	rowStr3 := fmt.Sprintf("%v %v", "4", "5")
+	r.Check(testkit.Rows(rowStr, rowStr1, rowStr2, rowStr3))
+	tk.MustExec("commit")
```

Comment:
Do we need to add some test about warnings?

Reply:
I'll add some tests for warnings in another pr.